shared/api/container_snapshot.go

```diff
@@ -44,6 +44,9 @@ type ContainerSnapshot struct {
 	LastUsedAt      time.Time                    `json:"last_used_at" yaml:"last_used_at"`
 	Name            string                       `json:"name" yaml:"name"`
 	Stateful        bool                         `json:"stateful" yaml:"stateful"`
+
+	// API extension: snapshot_disk_usage
+	Size int64 `json:"size" yaml:"size"`
```

Comment:
We're not adding new fields to the `container*` files, they're just there for backward compatibility.

Reply:
OK. I removed that part.